File: backend/src/main/java/org/jdk/project/dto/config/RegionDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class RegionDto {
  private Long id;
  private String regionName;
  private Long areaId;
  private String areaName;
  private String roomCode;
  private String deptName;
  private String coordinate;
  private Integer maxCapacity;
  private String isGuide;
  private String isShow;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/RegionDto.kt
package org.jdk.project.dto.config

data class RegionDto(
    var id: Long? = null,
    var regionName: String? = null,
    var areaId: Long? = null,
    var areaName: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var coordinate: String? = null,
    var maxCapacity: Int? = null,
    var isGuide: String? = null,
    var isShow: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
